core/templates/dev/head/pages/exploration_editor/editor_tab/AnswerGroupsCacheService.js

```diff
@@ -29,7 +29,9 @@ oppia.factory('AnswerGroupsCacheService', [function() {
       return _cache.hasOwnProperty(interactionId);
     },
     set: function(interactionId, answerGroups) {
-      _cache[interactionId] = angular.copy(answerGroups);
+      if (interactionId) {
```

Comment:
Why wouldn't there be an interactionId?

OK to keep this but maybe add "else throw Error('...')" at the bottom. And if you're aware of a case where there's no interaction ID, prevent it from calling set() in the first place.

Reply:
Done.